src/main/java/Todo.java

```diff
@@ -0,0 +1,10 @@
+public class Todo extends Task {
+    Todo(String task) {
```

Comment:
I like how you split up each type of task into different files, making the code really clean!